x/community/keeper/grpc_query.go

```diff
@@ -45,3 +46,33 @@ func (s queryServer) TotalBalance(
 		Pool: totalBalance,
 	}, nil
 }
+
+// AnnualizedRewards calculates the annualized rewards for the chain.
+// This method is backported from v0.25.x to allow for early migration.
+func (s queryServer) AnnualizedRewards(
+	c context.Context,
+	req *types.QueryAnnualizedRewardsRequest,
+) (*types.QueryAnnualizedRewardsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	bondDenom := s.keeper.stakingKeeper.BondDenom(ctx)
+
+	totalSupply := s.keeper.bankKeeper.GetSupply(ctx, bondDenom).Amount
+	totalBonded := s.keeper.stakingKeeper.TotalBondedTokens(ctx)
+	rewardsPerSecond := sdkmath.LegacyZeroDec() // always zero. this method is backported from v0.25.x to allow for early migration.
```

Comment:
rather than update the signature of `CalculateStakingAnnualPercentage`, i just hardcoded the rewardsPerSecond in this branch to zero.

Reply:
👍 great call setting that to zero here